src/main/java/com/blackducksoftware/integration/hub/alert/channel/slack/SlackChannel.java

```diff
@@ -0,0 +1,118 @@
+ * http://www.blackducksoftware.com/
+ *
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.alert.channel.slack;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.stereotype.Component;
+
+import com.blackducksoftware.integration.exception.IntegrationException;
+import com.blackducksoftware.integration.hub.alert.channel.DistributionChannel;
+import com.blackducksoftware.integration.hub.alert.channel.SupportedChannels;
+import com.blackducksoftware.integration.hub.alert.channel.rest.ChannelRestConnectionFactory;
+import com.blackducksoftware.integration.hub.alert.datasource.entity.SlackConfigEntity;
+import com.blackducksoftware.integration.hub.alert.datasource.repository.SlackRepository;
+import com.blackducksoftware.integration.hub.alert.digest.model.ProjectData;
+import com.blackducksoftware.integration.hub.rest.RestConnection;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import okhttp3.HttpUrl;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+@Component
+public class SlackChannel extends DistributionChannel<SlackEvent, SlackConfigEntity> {
+    private final static Logger logger = LoggerFactory.getLogger(SlackChannel.class);
+
+    @Autowired
+    public SlackChannel(final Gson gson, final SlackRepository slackRepository) {
+        super(gson, slackRepository, SlackEvent.class);
+    }
+
+    @Override
+    public void sendMessage(final SlackEvent event, final SlackConfigEntity config) {
+        final ProjectData projectData = event.getProjectData();
+        final String htmlMessage = createHtmlMessage(projectData);
+        sendMessage(htmlMessage, config);
+    }
+
+    @Override
+    public String testMessage(final SlackConfigEntity config) {
+        final String message = "Test from Alert application";
+        return String.valueOf(sendMessage(message, config));
+    }
+
+    private int sendMessage(final String htmlMessage, final SlackConfigEntity config) {
```

Comment:
I'd like to see an example of what a notification generated by the Hub looks like.